.ci/util.shlib

```diff
@@ -301,6 +301,30 @@ function UTIL_LOAD_CUSTOM_HOOK() {
         }
         popd > /dev/null || return 1
 	fi
+
+# $1: The repo name to parse
+# $2: The database URL
+function UTIL_PARSE_DATABASE() {
+    local repo="$1"
+    local db_url="$2"
+    local already_parsed=() # Prevent processing one pkgbase more than once
+
+    wget -q "${db_url}"
+    TEMP="$(mktemp -d)"
```

Comment:
should probably clean this up at the end of this function

Reply:
We can't clean this up, as we need to parse the changes afterwards. Cleaning up once being done with the run should be fine?